manifest_staging/charts/gatekeeper/README.md

```diff
@@ -37,3 +37,5 @@ template, please edit `kustomization.yaml` and `replacements.go` under that
 directory and then run `make manifests`. Your changes will show up in the
 `manifest_staging` directory and will be promoted to the root `charts` directory
 the next time a Gatekeeper release is cut.
+
+test
```

Comment:
mistake?

Reply:
added to trigger helm upgrade since it's a change in manifest_stating, will remove before merge